WordPress/Classes/Services/ReaderPostService.h

```diff
@@ -9,6 +9,9 @@ extern NSString * const ReaderPostServiceToggleSiteFollowingState;
 
 @interface ReaderPostService : LocalCoreDataService
 
+@property (nonatomic) NSString *nextPageHandle;
```

Comment:
We should copy the string. 

```suggestion
@property (copy, nonatomic) NSString *nextPageHandle;
```

Reply:
Done in 57305aaf43f93709228d56bd60c7a767f7f6f0cf